### ISSUE TITLE
Side bar links break on scroll

### ISSUE THREAD

**Issue by vitawood** [CameronNicolson/Leaf-It-To-Us#19]
Side bar sticks when you scroll but the links don't work

**Comment by CameronNicolson** [comment]
z-index property? [https://www.w3schools.com/cssref/pr_pos_z-index.asp](https://www.w3schools.com/cssref/pr_pos_z-index.asp)
```
#sidebar {
    position: fixed;
    z-index: 1;
    background-color: #dbd5c9;
    width: 20%;
    padding: 10% 0% 10% 5%;
    box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
```

**Comment by CameronNicolson** [comment]
I committed a possible fix just need for you to test it 2cfe5ea154cdccaf1007473bafecc8d0d99ce0a3

**Comment by vitawood** [comment]
seems to be good on catalog page, that's the only one consistently scrollable right now so let's call it fixed
